_includes/v3.8/release-notes/v3.8.5-release-notes.md

```diff
@@ -0,0 +1,7 @@
+06 Dec 2019
+
+#### Other changes
+
+ - Timeouts added to Calico CNI calls [cni-plugin #816](https://github.com/projectcalico/cni-plugin/pull/816) (@rafaelvanoni)
+ - Add bird/bird6 liveness checks [node #374](https://github.com/projectcalico/node/pull/374) (@asincu)
+ - Backport error handling improvements [pod2daemon #31](https://github.com/projectcalico/pod2daemon/pull/31) (@fasaxc)
```

Comment:
Would be good to have a more user-facing release note for this. "Error handling improvements" is pretty vague.